packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -318,6 +327,10 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         this.throwOnTombstoneUsage =
             this.mc.config.getBoolean(throwOnTombstoneUsageKey) === true &&
             this.clientDetails.type !== summarizerClientType;
+
+        // By default, a data store can log maximum 100 local changes telemetry in summarizer.
+        this.localChangesTelemetryThreshold =
+            this.mc.config.getNumber("Fluid.Telemetry.LocalChangesTelemetryThreshold") ?? 100;
```

Comment:
100 seems like really high number. Are you sure that's good starting point?

Reply:
I am not sure what is a good number. I don't think data stores in summarizer will send lot of ops so 100 may be okay. Do you have any suggestion on what can be a good number here? 10?

Reply:
I believe this will result (in worst case) in 100 events per offending data store, right?
I think we should consider the probability of extra events providing us data that we did not know with previous events (like new component type). I think that probability is close to zero after first 10 events - mostly because we are likely to get that info either way from one or another session (i.e. that component is likely to be in first 10 in some session).

Reply:
Changed it to 10.